__tests__/app.test.js

```diff
@@ -312,4 +312,117 @@ describe("GET comments by review ID", () => {
                 });
         });
     });
+});
+
+describe("POST new comment", () => {
+    describe("Happy path :)", () => {
+        let testComment = {
+            username: "philippaclaire9",
+            body: "Where is my head?"
+        };
+
+        test("When a valid comment is added it is returned as an object", () => {
+            return request(app)
+                .post("/api/reviews/1/comments")
+                .send(testComment)
+                .expect(201)
+                .then(({ body }) => {
+                    const newComment = body.comment;
+                    expect(newComment.author).toEqual(testComment.username);
```

Comment:
Please check the whole comment looks as we expect, aka check the comment has the right comment_id, review_id, body, created_at etc.